README.md

```diff
@@ -68,7 +69,18 @@ The commandArgs should contain any commandline properties and options that would
 The options should contain a files object, as well as any of the following options:
 * __usePods__ _(boolean)_: Sets up `.ember-cli` file with `"usePods": true`. Default: false
 * __podModulePrefix__ _(boolean)_: Sets up `config/environment.js` with 'app/pods' as the `podModulePrefix`. Default: false
+* __skipInit__ _(boolean)_: Skips the temporary project init step for situations where the project has already been setup. Most commonly used when generating inside the `afterGenerate` hook.
+
 * __target__ _(string)_: Defines the type of project to setup for the test. Recognized values: __'app'__, __'addon'__, __'inRepoAddon'__
+* __assertions__ _(array)_: An array of custom assertions to make after the blueprint has been generated, but before it has been destroyed. Generally you would want to use helper functions that return a Chai assertion.
+Example:
+
+  ```js
+  assertions: [
+    expect(foo).to.be.ok
+  ]
```

Comment:
hmm, I don't think that will work because `expect(foo).to.be.ok` will be evaluated directly while building the `assertions` array